src/scripts/entrypoint.sh

```diff
@@ -121,10 +97,10 @@ setup_cron() {
     "BASH_ENV=/env.sh" \
     "${SCRIPT_PATH}" \
     "${LOG_LOCATION}" \
-    > "/etc/cron.d/${CRON_NAME}"
-  echo "" >> "/etc/cron.d/${CRON_NAME}"
+    | sudo tee "/etc/cron.d/${CRON_NAME}"
+  echo "" | sudo tee -a "/etc/cron.d/${CRON_NAME}"
```

Comment:
@FooDeas this line sets the cron script as root, the gosu should still work for the crons. ill setup some crons for tonight and let em run

Reply:
Right. I thought from the wrong user context.

Reply:
You were right on the call out! im reworking the cron system rn

Reply:
Interesting. I was confused by the "**sudo** tee" statements writing the cron files. Based on this, I wrongly concluded that it is now in the Steam user context. After your hint, I also would have expected that there would be no problems. 😮